### PR TITLE
add note about upload key collisions

### PR DESCRIPTION
**Pull request by mifi**
due to reserved form post fields
see https://transloadit.com/docs/api/assemblies-assembly-id-replay-post/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +154,8 @@ You can provide the following keys inside the `options` object:
 - `chunkSize` - (for uploads) a number indicating the maximum size of a tus `PATCH` request body in bytes. Default to `Infinity` for file uploads and 50MB for streams of unknown length. See [tus-js-client](https://github.com/tus/tus-js-client/blob/master/docs/api.md#chunksize).
 - `uploadConcurrency` - Maximum number of concurrent tus file uploads to occur at any given time (default 10.)
 
+ **NOTE**: Make sure the key in `files` and `uploads` is not one of `signature`, `params` or `max_size`.
```
Comment (kvz) [inline README.md:157]: What about throwing an error when someone tries this?

Comment (mifi) [reply README.md:157]: I thought about that, but then if someone builds an app where the `key` is a dynamic value provided by some other system (or user), then everything will look fine when developing and testing, but suddenly crash in production on a sunday morning.

Comment (kvz) [reply README.md:157]: we could add a todo for the next major maybe?

Comment (mifi) [reply README.md:157]: As far as I understand, this is a limitation in the api2 `create assembly` endpoint. Even if we make a new `create assembly` endpoint v2, like `POST https://api2.transloadit.com/v2/assemblies`, I'm not sure how to make an API that allows:
1. upload arbitrary number of files with arbitrary form keys
2. post other data using the same form with keys `signature`, `params` and `max_size`

...in the same request. I think those two things are not possible to combine

**Pull request merged**